src/commands/deploy.ts

```diff
@@ -65,26 +66,31 @@ export default class Deploy extends Command {
         const data = new FormData();
         data.append('app', fs.createReadStream(fd.mergeWithFolder(zipName)));
 
-        await this.asyncSubmitData(data, flags);
+        await this.asyncSubmitData(data, flags, fd);
 
         cli.action.stop('deployed!');
     }
 
     // tslint:disable:promise-function-async
-    private async asyncSubmitData(data: FormData, flags: { [key: string]: any }): Promise<void> {
+    private async asyncSubmitData(data: FormData, flags: { [key: string]: any }, fd: FolderDetails): Promise<void> {
         const authResult = await fetch(this.normalizeUrl(flags.url, '/api/v1/login'), {
             method: 'POST',
             headers: {
                 'Content-Type': 'application/json',
             },
-            body: JSON.stringify({ username: flags.username, password: flags.username }),
+            body: JSON.stringify({ username: flags.username, password: flags.password }),
```

Comment:
:see_no_evil: whoopies

Reply:
Damn!! That's why I don't code anymore 😂

Reply:
LOL

Reply:
No no, I was talking about myself :stuck_out_tongue_winking_eye:

Reply:
Yeah, I realized after :P then I deleted... hahahha

Reply:
LOL